easycv/core/evaluation/coco_evaluation.py

```diff
@@ -306,20 +306,13 @@ def _evaluate_impl(self, prediction_dict, groundtruth_dict):
                 detection_classes = detection_classes.cpu().numpy().astype(
                     np.int32)
 
-            whwh = np.array([width, height, width, height], dtype=np.float32)
-
-            # target boxes
-            if len(gt_boxes) > 0:
-                gt_boxes_absolute = xywh2xyxy(gt_boxes) * whwh
-            else:
-                continue
             if groundtruth_is_crowd_list is None:
                 groundtruth_is_crowd = None
             else:
                 groundtruth_is_crowd = groundtruth_is_crowd_list[idx]
 
             groundtruth_dict = {
-                'groundtruth_boxes': gt_boxes_absolute,
+                'groundtruth_boxes': gt_boxes,
```

Comment:
input gt_boxes should be absolute coordiante?  add this requirement to docstr of this function

Reply:
done!